DashSync/Models/Platform/Transitions/BlockchainIdentity/DSBlockchainIdentityUpdateTransition.h

```diff
@@ -0,0 +1,40 @@
+//
```

Comment:
Seems unused

Reply:
Yep, unused for now, but will use it later.